keyboards/alectardy98/alpinenoon/keymaps/default/keymap.c

```diff
@@ -0,0 +1,49 @@
+/* Copyright 2022 Alectardy98
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+enum _layer {
+  _BASE,
+  _FN
+};
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+[_BASE] = LAYOUT(
+    KC_F1,  KC_F2,   KC_ESC,    KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0, KC_MINS,  KC_EQL, KC_GRV, KC_BSPC, KC_PSCR, \
+    KC_F3,  KC_F4,   KC_TAB,    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P, KC_LBRC, KC_RBRC, KC_BSLS,  KC_DEL,  \
+    KC_F5,  KC_F6,  KC_CAPS,    KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L, KC_SCLN, KC_QUOT, KC_BSLS,  KC_ENT, KC_HOME, \
+    KC_F7,  KC_F8,  KC_LSFT, KC_NUBS,    KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M, KC_COMM,  KC_DOT, KC_SLSH, KC_RSFT,   KC_UP,  KC_END,  \
+    KC_F9, KC_F10, KC_LCTL,  KC_LGUI, KC_LALT,                             KC_SPC,                   KC_RALT, KC_RGUI,          KC_LEFT, KC_DOWN, KC_RGHT),
+
+[_FN] = LAYOUT(
```

Comment:
- `LAYOUT` -> `LAYOUT_all`
- remove trailing backslashes (they're not needed at all in keymaps)

```suggestion
[_BASE] = LAYOUT_all(
    KC_F1,  KC_F2,   KC_ESC,    KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0, KC_MINS,  KC_EQL, KC_GRV, KC_BSPC, KC_PSCR,
    KC_F3,  KC_F4,   KC_TAB,    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P, KC_LBRC, KC_RBRC, KC_BSLS,  KC_DEL,
    KC_F5,  KC_F6,  KC_CAPS,    KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L, KC_SCLN, KC_QUOT, KC_BSLS,  KC_ENT, KC_HOME,
    KC_F7,  KC_F8,  KC_LSFT, KC_NUBS,    KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M, KC_COMM,  KC_DOT, KC_SLSH, KC_RSFT,   KC_UP,  KC_END,
    KC_F9, KC_F10, KC_LCTL,  KC_LGUI, KC_LALT,                             KC_SPC,                   KC_RALT, KC_RGUI,          KC_LEFT, KC_DOWN, KC_RGHT),

[_FN] = LAYOUT_all(
```